zppy/tc_analysis.py

```diff
@@ -64,3 +69,8 @@ def tc_analysis(config, scriptDir):
                     # Update status file
                     with open(statusFile, "w") as f:
                         f.write("WAITING %d\n" % (jobid))
+
+                # Note that this line should still be executed even if jobid == -1
+                # The later tc_analysis tasks still depend on this task (and thus will also fail).
+                # Add to the dependency list
+                dependencies.append(statusFile)
```

Comment:
@chengzhuzhang I realized we can do what `mpas_analysis` does to force `tc_analysis` to run sequentially. I'm testing this now, but I think this should at least provide a work-around. If the tests work, I will squash and merge this PR.

I've documented the shared library error in #180, as it would be good to run multiple `year_sets` in parallel.

Reply:
Sounds good. I think it's a reasonable temporal workaround.